tests/utils/walletConfig.test.js

```
@@ -0,0 +1,148 @@
+});
+
+describe("getConnectorDiscovery function", () => {
+  beforeEach(() => {
+    global.navigator = {
+      userAgent:
+        "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/58.0.3029.110 Safari/537.36",
+    };
+  });
+
+  afterEach(() => {
+    delete global.navigator;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)